### ISSUE TITLE
GitHub mirror: lock conversations on GitHub commits

### ISSUE THREAD

**Issue by ianlancetaylor** [golang/go#43745]
The Go project mirrors all commits from [Gerrit](https://go.googlesource.com/go/+/refs/heads/master) to [GitHub](https://github.com/golang/go/commits/master).  The GitHub commits point back to the associated Gerrit code review.  Despite that, people sometimes comment on the GitHub commits.  Very few people see those comments, and they don't help the overall project.

I don't know what does this mirroring, but is it possible for that process to lock the conversation for the GitHub commit?  There is a "Lock conversation" link on the commit, so it seems that it ought to be possible to automate the locking process.

CC @golang/release

**Comment by dmitshur** [comment]
The mirroring is done by [`gitmirror`](https://pkg.go.dev/golang.org/x/build/cmd/gitmirror). Unfortunately, we are still waiting on GitHub to expose the ability to lock conversations on commits via an API (it's possible to lock issues and PRs, but not commits), see  https://github.com/golang/go/issues/25450#issuecomment-412196688. I just checked again, I don't think there's been a change on GitHub's side. I'll close this as duplicate of that issue.

**Comment by ianlancetaylor** [comment]
Thanks.
